README.md

```diff
@@ -46,50 +57,55 @@ $ sudo apt-get update
 $ sudo apt-get install cabal-install-1.20 ghc-7.8.3 happy-1.19.4 alex-3.1.3
 ```
 
-Then add
-`~/.cabal/bin:/opt/cabal/1.20/bin:/opt/ghc/7.8.3/bin:/opt/happy/1.19.4/bin:/opt/alex/3.1.3/bin` to your PATH (bash_profile, zshrc, bashrc, etc)
+Then add the following to your `$PATH` (bash\_profile, zshrc, bashrc, etc):
+
+- `~/.cabal/bin`
+- `/opt/cabal/1.20/bin`
+- `/opt/ghc/7.8.3/bin`
+- `/opt/happy/1.19.4/bin`
+- `/opt/alex/3.1.3/bin`
```

Comment:
It was too long to fit in the width of a normal page. I'll try and see it it does now, as I've changed the pages to landscape mode.